README.md

```diff
@@ -163,6 +163,7 @@ Safe Logging can be found at [github.com/palantir/safe-logging](https://github.c
 - `DangerousCompletableFutureUsage`: Disallow CompletableFuture asynchronous operations without an Executor.
 - `NonComparableStreamSort`: Stream.sorted() should only be called on streams of Comparable types.
 - `DangerousStringInternUsage`: Disallow String.intern() invocations in favor of more predictable, scalable alternatives.
+- `OptionalOrElseThrowThrows`: Optional.orElseThrow argument must return an exception, not throw one.
```

Comment:
could we be more precise and say this only supports lambda arguments?

Reply:
I think that's an implementation detail, it's the most common case, and the one that I implemented this to support, but it wouldn't be unreasonable to eventually add support for anonymous classes (though I don't think it's worthwhile).